### DISCUSSION TITLE
Docker and ssh keys support?

### DISCUSSION THREAD

**Discussion by zinwelzl** [cockpit-project/cockpit#18241] [Q&A] [answered]
Hi.

I can see podman plugin, but what about docker?
I don't see any docker images/containers/etc

How can I set up ssh keys for login?

**Comment by jelly** [comment]
We only have a podman plugin no docker plugin, but the podman plugin should support containers from dockerhub fine..

What do you mean with "set up ssh keys for login"? As in login to cockpit with ssh?

**Comment by zinwelzl** [comment]
I have docker images/containers setup, and podman don't "see" them, in terminal or cockpit.

ssh keys login to ssh server.
I can use ssh-keygen and ssh-copy-id in terminal.
Can't find how do to it in cockpit?

**Comment by jelly** [comment]
podman uses different images then docker so from that aspect it is not compatible, it is compatible to use the same images from an image registry.

**Comment by chabad360** [comment, marked as answer]
I have forked the cockpit-podman plugin and [ported it to docker](https://github.com/chabad360/cockpit-docker).
